csl.rnc

```diff
@@ -13,8 +13,9 @@ dc:creator [ "Simon Kornblith" ]
 bibo:editor [ "Frank Bennett" ]
 bibo:editor [ "Rintze Zelle" ]
 dc:rights [
-  "Copyright 2007-2012 by Frank Bennett, Bruce D'Arcus, Simon Kornblith, and Rintze Zelle. Permission to freely use, copy and distribute."
+  "Copyright 2007-2017 Frank Bennett, Bruce D'Arcus, Simon Kornblith, and Rintze Zelle"
```

Comment:
Reminder to update to 2018 or even removing these dates if they're more trouble than they're worth.